rust/lance-io/src/object_store.rs

```diff
@@ -308,6 +308,11 @@ pub struct ObjectStoreParams {
     pub aws_credentials: Option<AwsCredentialProvider>,
     pub object_store_wrapper: Option<Arc<dyn WrappingObjectStore>>,
     pub storage_options: Option<HashMap<String, String>>,
+    /// Use constant size upload parts for multipart uploads. Only necessary
+    /// for Cloudflare R2, which doesn't support variable size parts. When this
+    /// is false, max upload size is 2.5TB. When this is true, the max size is
```

Comment:
Saw in PR body you mentioned it's 2.5GB, guess 2.5TB is the right one?

Reply:
Yeah thanks for the correction.